src/pages/[guild]/index.tsx

```diff
@@ -136,6 +140,29 @@ const GuildPage = (): JSX.Element => {
         textColor={textColor}
         description={description}
         showLayoutDescription
+        infoSection={
+          Object.keys(socialLinks ?? {}).length && (
+            <Wrap w="full" spacing={3} mb={8}>
+              {Object.entries(socialLinks).map(([type, link]) => (
+                <HStack key={type} spacing={1.5}>
+                  <SocialIcon type={type as SocialLinkKey} size="sm" />
+                  <Link
+                    href={link}
+                    isExternal
+                    fontSize="sm"
+                    fontWeight="semibold"
+                    color={textColor}
+                  >
+                    {link
+                      .replace("http://", "")
+                      .replace("https://", "")
+                      .replace("www.", "")}
```

Comment:
This looks dangerous. Consider doing a regex replace if this is meant to strip a leading `www.`